docs/technical-documentation.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Technical documentation
+category: Technical documentation
+---
+
+# Architecture
+
+## Micro-services architecture
+
+Here is an overview of PowSyBl micro-services architecture and corresponding repositories:
+
+<object type="image/svg+xml" data="./architecture/images/micro-services-architecture.svg" >
+</object>
+
+### Case server
+
+This services is responsible from raw network data storage. It relies on a file system to store its data and allows uploading and fetching network data for any network data format supported by PowSyBl (CGMES, UCTE, XIIDM, etc).
```

Comment:
Typo: this servic**e** is
Change a little bit *fetching network data for any network data* to limit the usage of *data*. Maybe *fetching network for any supported format by PowSyBl*

Reply:
Done